libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -834,6 +835,17 @@ AP_InertialSensor::detect_backends(void)
     } else {
         hal.console->printf("aero: onboard IMU not detected\n");
     }
+#elif HAL_INS_DEFAULT == HAL_INS_RST
+    AP_InertialSensor_Backend *backend = AP_InertialSensor_RST::probe(*this, hal.spi->get_device(HAL_INS_RST_G_NAME), 
```

Comment:
here you use a macro, but in the spidesc you use the name directly. Either use the macro there as well or just remove the macro and use the name directly like is done for other boards.

Reply:
In the spidesc I use the name directly because other boards all use name macro. In AP_InertialSensor.cpp, I find AERO,BBBMINI and other linux boards use HAL_INS_DEFAULT macro, so I did this just like them, I think it will be wierd to use name macro here.What do you think?

Reply:
No... I was talking about  `HAL_INS_RST_G_NAME` vs just using the name.